warp10/src/main/java/io/warp10/script/ext/token/TokenWarpScriptExtension.java

```diff
@@ -35,7 +35,7 @@
    * Name of configuration key with OSS wrapped token secret
```

Comment:
It's actually the name of the secret in the key store, not in the configuration.